[WIP] Add IAST benchmarks (#3193)
DataDog/dd-trace-js: benchmark/sirun/appsec-iast/README.md

This creates 150 HTTP requests from client to server.

The variants are:

> - control tracer in no vulnerable endpoint without iast 
> - tracer in no vulnerable endpoint with iast active and default configuration
> - tracer in no vulnerable endpoint with iast active and sampling 100
> - control tracer in vulnerable endpoint without iast
> - tracer in vulnerable endpoint with iast active and default configuration
> - tracer in vulnerable endpoint with iast active and sampling 100

Reviewer: ```suggestion
- control tracer with non vulnerable endpoint without iast 
- tracer with non vulnerable endpoint with iast active and default configuration
- tracer with non vulnerable endpoint with iast active and sampling 100
- control tracer with vulnerable endpoint without iast
- tracer with vulnerable endpoint with iast active and default configuration
- tracer with vulnerable endpoint with iast active and sampling 100
```
